### PR TITLE
Travis setup for 5 nodes

### PR DESCRIPTION
**Pull request by AnkshitJain**
Makes travis testing for the five node setup. The check for request status page has been disabled.

### CODE REVIEW

**Comment by AnkshitJain** [comment]
Since a skip TODO has been introduced in place of the test **load Autolab status page** in website load tests, code climate shows 1 issue to fix. This will be corrected in the next PR.

File: tests/functional_tests/website-load.bats [hunk 1]
```diff
@@ -25,10 +25,11 @@ teardown() {
 }
 
 @test "load Autolab status page" {
-  curl -s --ipv4 -k https://127.0.0.1:9000/status -o "$BATS_TMPDIR/website-load-tests/status.txt"
-  cmp "$BATS_TMPDIR/website-load-tests/status.txt" data/autolab-start/status.txt
-  result=$?
-  [ "$result" -eq 0 ]
+  skip "TODO"
```
Comment (prasadtalasila) [inline tests/functional_tests/website-load.bats]: shouldn't this be before line-28?

File: tests/functional_tests/missing_files.bats [hunk 2]
```diff
@@ -33,8 +48,16 @@ teardown() {
...
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_4/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_5/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_1/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_2/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_3/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_4/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/execution_node_5/extract_run.sh
   node submit.js -i 2015A7PS006G -l lab1 --lang=java --host='localhost:9000' > \
```
Comment (prasadtalasila) [inline tests/functional_tests/missing_files.bats]: changes required here to exercise all the five execution nodes. As you mentioned, we need to have tests for cases illustrated in issue #206.

File: tests/functional_tests/missing_files.bats [hunk 3]
```diff
@@ -33,8 +48,16 @@ teardown() {
 
 #Note : The output of this test will differ if any changes are made to docs/examples/unit_tests/execute.sh
 @test "run no test info found" {
-  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/extract_run.sh
-  sed -i '/bash execute.sh "$language"/ a cp ./results/log.txt /tmp/missing-files/no-test-info-found-log.txt' ../../execution_nodes/extract_run.sh
+  sed -i '/bash execute.sh "$language"/ i rm -f ./test_info.txt' ../../execution_nodes/execution_node_1/extract_run.sh
```
Comment (prasadtalasila) [inline tests/functional_tests/missing_files.bats]: There seems to be too much of repetition here? would a shell function help?

Comment (AnkshitJain) [reply tests/functional_tests/missing_files.bats]: Will it work if we keep only one copy for extract_run.sh for all the execution nodes in the parent directory execution_nodes? We reduce the duplication of that script, and to make matters simple, we can add another parameter to the extract_run.sh script, which specifies from which node the request is coming from, so the evaluation can happen inside the specific execution node directory.

File: tests/functional_tests/website-load.bats (hunk 1 above)
Comment (AnkshitJain) [reply tests/functional_tests/website-load.bats]: My bad. I'll update it.

**Comment by AnkshitJain** [comment]
The updated status message will come in handy when we are testing for the concurrent working  of the execution nodes. In the status message, if we add a parameter, which denotes the node_queue_length, we can identify how many nodes are being occupied at the point the status message was requested. The job_queue_length along with the node queue length can be useful in determining the current work being processed and pending on the system, rather than just the pending work.

**Comment by AnkshitJain** [comment]
@prasadtalasila Please have a look at the new commit which includes the test.

**Pull request closed without merging**